### PR TITLE
Added Ask.com support

### PR DESCRIPTION
**Pull request by sayak119**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -26,4 +26,4 @@
         "postinstall": "bower install"
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (AnshulMalik) [inline app.json]: We should keep this empty line

File: app/server.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from pymongo import MongoClient
 from dicttoxml import dicttoxml
 from xml.dom.minidom import parseString
-import json,os
+import json,os,sys
```
Comment (AnshulMalik) [inline app/server.py]: Umm, where is this module used?

File: app/server.py [hunk 3]
```diff
@@ -75,4 +75,4 @@ def set_header(r):
 
 if __name__ == '__main__':
     app.debug = True
-    app.run(host='0.0.0.0', port=(int)(os.environ.get('PORT', 7001)), debug=True)
\ No newline at end of file
+    app.run(host='0.0.0.0', port=(int)(os.environ.get('PORT', 7001)), debug=True)
```
Comment (AnshulMalik) [inline app/server.py]: Let's keep the empty line

**Comment by sayak119** [comment]
Making a new pull request [https://github.com/fossasia/query-server/pull/148](url)

**Pull request closed without merging**